CHANGELOG.md

```diff
@@ -61,6 +61,11 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#70](https://github.com/realm/SwiftLint/issues/70)
 
+* Add new `prefer_key_path` rule that triggers when a trailing closure on a function 
```

Comment:
Should it be mentioned that the rule is opt-in or disabled by default?

Reply:
No, that's not necessary as it can be looked up in the [Rule Directory](https://realm.github.io/SwiftLint/rule-directory.html).